src/go/cmd/http-relay-server/broker.go

```diff
@@ -209,13 +232,22 @@ func (r *broker) SendResponse(resp *pb.HttpResponse) error {
 	duration := time.Since(pr.startTime).Seconds()
 	pr.responseStream <- resp
 	r.m.Unlock()
-	brokerRequests.WithLabelValues("server_response", backendName).Inc()
-	brokerResponseDurations.WithLabelValues("server_response", backendName).Observe(duration)
-	log.Printf("[%s] Delivered response to client (%d bytes), elapsed %.3fs", id, len(resp.Body), duration)
+	brokerRequests.WithLabelValues("server_response", backendName, pr.requestPath).Inc()
+	brokerResponseDurations.WithLabelValues("server_response", backendName, pr.requestPath).Observe(duration)
 	if resp.GetEof() {
 		close(pr.responseStream)
+		backendDuration := (time.Duration(resp.GetBackendDurationMs()) * time.Millisecond).Seconds()
```

Comment:
this first creates a Duration in a wrong unit (BackendDurationMs() is in Milisecond but Duration is in ns) and fixes it afterwards. please get the unit right before converting to time.Duration.

Reply:
Yes, that looks weird, but I can't write:
```
time.Duration(resp.GetBackendDurationMs() * time.Millisecond))
```
as the compiler complains about:
```
invalid operation: resp.GetBackendDurationMs() * time.Millisecond (mismatched types int64 and time.Duration)
```
Any better idea?`